src/Raven.Studio/typescript/viewmodels/database/documents/editDocument.ts

```diff
@@ -569,8 +569,9 @@ class editDocument extends viewModelBase {
         return collectionForNewDocument + "/";
     }
 
-    toClipboard() {
-        copyToClipboard.copy(this.documentText(), "Document has been copied to clipboard");
+    toClipboard(copyId: boolean = false) {
```

Comment:
maybe we should split this into 2 methods:
copyDocumentIdToClipboard
copyDocumentBodyToClipboard?

Reply:
That was exactly my first thought - and then I changed to this because I thought you would prefer it united :)

Reply:
done